REQUIRE

```diff
@@ -1 +1,2 @@
 julia 0.5
+Compat 0.18
```

Comment:
and given this package is 0.5-only, shouldn't be needed at all? or was there something else not changed in this patch that was solved just by using Compat?